### DISCUSSION TITLE
quartz/

### DISCUSSION THREAD

**Discussion by giscus** [Signal-K/Unity-Demo#26] [General]
# quartz/

🌱 host your own second brain and digital garden

https://app.skinetics.tech/quartz/

**Comment by Gizmotronn** [comment]
[![](https://images.unsplash.com/photo-1595496115800-e35438b7d3b8?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1332&q=80)](https://docs.skinetics.tech/)
